.github/workflows/ci.yml

```diff
@@ -189,16 +189,18 @@ jobs:
             - { target: thumbv7em-none-eabihf,            os: ubuntu-latest,                    std: 1 }
             - { target: thumbv7m-none-eabi,               os: ubuntu-latest,                    std: 1 }
 
-  docker-image:
-    name: docker-image (${{ matrix.pretty }})
+  build:
+    name: target (${{ matrix.pretty }},${{ matrix.os }})
```

Comment:
Renamed the job name, `docker-image` doesn't make much sense since this also creates binary artifacts